lib/unwrappr/lock_file_diff.rb

```diff
@@ -63,7 +63,7 @@ def extract_gem_and_change_type(line)
       # '+    websocket-driver (0.6.5)'
       # Careful not to match this (note the wider indent):
       # '+      websocket-extensions (>= 0.1.0)'
-      pattern = /^(?<change_type>[\+\-])    (?<gem_name>[\w-]+) \(\d/
+      pattern = /^(?<change_type>[\+\-])    (?<gem_name>[\S-]+) \(\d/
```

Comment:
Rather than expecting the gem name to consist of only _word_ characters (`[a-zA-Z0-9_]`), expect it to be a string of non-whitespace characters (`[^ \t\r\n\f\v]`).